waiter/src/waiter/handler.clj

```diff
@@ -891,6 +891,21 @@
        :leader-id (leader-id-fn)})
     router-id request))
 
+(defn get-tokens-watch-maintainer-state
+  "Using the query-state-fn, pass include-flags to get the tokens-watch-maintainer-state and return
+  streaming response"
+  [router-id query-state-fn request]
+  (try
+    (let [{:strs [include]} (-> request ru/query-params-request :query-params)
+          include-flags (if (string? include) #{include} (set include))
+          state (if (nil? include)
+                  (query-state-fn)
+                  (query-state-fn include-flags))]
```

Comment:
We should not need to support two variants of the `query-state-fn`. We can support one variant: `(query-state-fn include-flags)`

Reply:
do we want the default behavior (include-flags is nil) to provide the entire state?